R/navbar-menu.R

```diff
@@ -64,7 +74,8 @@ navbar_html <- function(x, path_depth = 0L, menu_depth = 0L, side = c("left", "r
     heading = navbar_html_heading(x),
     link = navbar_html_link(x, menu_depth = menu_depth),
     separator = navbar_html_separator(),
-    search = navbar_html_search(x, path_depth = path_depth)
+    search = navbar_html_search(x, path_depth = path_depth),
+    lightswitch = navbar_html_lightswitch()
```

Comment:
Is this outdated? I can't find `navbar_html_lightswitch()`

Reply:
Oops, yes, I forgot to remove that vestige.